uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -376,21 +378,49 @@ sub identify_scan_db {
         Model         => $fileref->getParameter('manufacturer_model_name'),
         Serial_number => $fileref->getParameter('device_serial_number'),
         Software      => $fileref->getParameter('software_versions')
-	});
+    });
     
     # default ScannerID to 0 if we have no better clue.
     my $ScannerID = @$resultsRef> 0 ? $resultsRef->[0]->{'ID'} : 0;
     
-    # get the list of protocols for a site their scanner and subproject
-    $query = "SELECT *
-              FROM mri_protocol
-              WHERE
-             (Center_name='$psc' AND ScannerID='$ScannerID')
-              OR ((Center_name='ZZZZ' OR Center_name='AAAA') AND ScannerID='0')
-              ORDER BY Center_name ASC, ScannerID DESC";
+    #===========================================================#
+    # Get the list of lines in the mri_protocol table that      #
+    # apply to the given scan based on the center name, project # 
+    # ID, subproject ID and visit label                         #
+    #===========================================================#
+    ($query = <<QUERY) =~ s/\n/ /gm;
+    SELECT mp.Scan_type, mp.ScannerID, mp.Center_name, 
+           mp.TR_range, mp.TE_range, mp.TI_range, mp.slice_thickness_range,
```

Comment:
I think something went wrong with the rebase cause there are no more `*_range` fields. They all have been replaced by `*_min` `*_max`. Can you please make sure to have the latest code for that function?